modules/ossm-control-plane-operatorhub-deploy.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/service_mesh_install/installing-ossm.adoc
+
+[id="ossm-control-plane-operatorhub-deploy_{context}"]
+= Deploying the control plane using OperatorHub
+
+Follow this procedure to deploy the {ProductName} control plane from OperatorHub.
+
+.Prerequisites
+
+* An account with cluster administrator access.
```

Comment:
Cluster Admin privileges aren't needed here, only maistra-admin role (which grants permissions to create/update/delete controlplane CRs)

Reply:
Where in the process is the maistra-admin role created?  We should document granting the role to users after it's created.